python_course_1604/class_01/exercize_01_hello_world.py

```diff
@@ -5,3 +5,5 @@
 '''
 
 # Write a line that prints 'Hello world!'
+
+print ('Hello world! ')
```

Comment:
There is a space at the end of the string that you should remove to match specs.